### ISSUE TITLE
Flexicubes suddenly collapses in some intermediate training steps

### ISSUE THREAD

**Issue by bluestyle97** [nv-tlabs/FlexiCubes#19]
Hi, thanks for your excellent work! We are training a image-to-3D reconstruction model using flexicubes as the 3D representation. We query MLPs to obtain the SDFs, colors, deformations and weights, and use Flexicubes to extract the surface. We adopt image-based supervisions to train the network, such as RGB and mask errors. Besides, we also use the regularizers of Flexicubes refering to the implementation of Get3D (including the SDF regularization loss, L_dev and weights regularization loss). However, we found that flexicubes always suddenly collapses at some intermediate training steps and starts to generate floating surfaces. The following picture visualizes such artifacts:

![image](https://github.com/nv-tlabs/FlexiCubes/assets/20635237/f79839f4-73db-4245-99c0-51aadf5df7c8)

We have tried adjusting the learning rate but still cannot solve the problem. Could you please give some suggestions on the reason of this problem and how to solve it?
